### ISSUE TITLE
Pool acquireTimeoutMillis cannot be set higher than 60s

### ISSUE THREAD

**Issue by MattGson** [knex/knex#5379]
# Environment

Knex version: 2.3.0
Database + version: Postgres 14

# Bug

1. Explain what kind of behaviour you are getting and how you think it should do

When setting `acquireTimeoutMillis` > 60_000 (60s) the timeout still occurs after only 60s has elapsed.

There is currently a workaround by setting `acquireConnectionTimeout` as well.

2. Error message

```
TimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
```

3. Reduced test code

```ts
    let db = knex({
		pool: {
			min: 1,
			max:1,		
			acquireTimeoutMillis: 1_200_000,
		}
    });

// with workaround

    let db = knex({
                acquireConnectionTimeout: 1_200_000,
		pool: {
			min: 1,
			max:1,		
			acquireTimeoutMillis: 1_200_000,
		}
    });
	
```

Created PR here #5377

**Comment by apnerve** [comment]
@MattGson Can you close this issue as your PR has already been merged?

**Comment by MattGson** [comment]
closed by #5377
